Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/worker/PCRTestResultRetrievalWorker.kt

```diff
@@ -7,34 +7,33 @@ import dagger.assisted.Assisted
 import dagger.assisted.AssistedFactory
 import dagger.assisted.AssistedInject
 import de.rki.coronawarnapp.coronatest.CoronaTestRepository
-import de.rki.coronawarnapp.coronatest.execution.TestResultScheduler
 import de.rki.coronawarnapp.coronatest.latestPCRT
 import de.rki.coronawarnapp.coronatest.server.CoronaTestResult
 import de.rki.coronawarnapp.coronatest.type.CoronaTest
 import de.rki.coronawarnapp.coronatest.type.pcr.PCRCoronaTest
+import de.rki.coronawarnapp.coronatest.worker.execution.PCRTestResultScheduler
 import de.rki.coronawarnapp.notification.GeneralNotifications
 import de.rki.coronawarnapp.notification.NotificationConstants
 import de.rki.coronawarnapp.notification.PCRTestResultAvailableNotificationService
 import de.rki.coronawarnapp.util.TimeStamper
 import de.rki.coronawarnapp.util.worker.InjectedWorkerFactory
+import de.rki.coronawarnapp.worker.BackgroundConstants
 import kotlinx.coroutines.flow.first
 import org.joda.time.Duration
 import org.joda.time.Instant
 import timber.log.Timber
 
 /**
- * Diagnosis test result retrieval by periodic polling
- *
- * @see BackgroundWorkScheduler
+ * RAT retrieval by periodic polling
  */
-class DiagnosisTestResultRetrievalPeriodicWorker @AssistedInject constructor(
+class PCRTestResultRetrievalWorker @AssistedInject constructor(
```

Comment:
Reminds me a bit of the wohle event/presence tracing.
Let's call it `PCRResultRetrieval...` and `RAResultRetrieval`,

`PCRT` and `RAT` or `PCR` and `RA`.

I think we mostly have `PCR` and `RA` now?

Reply:
zubat attacks with confusion... shiggy is now confused.

Reply:
renamed it